### PR TITLE
[BEAM-2937] Disable combiner lifting optimization in python DataflowRunner for fnapi

### PR DESCRIPTION
**Pull request by aaltay**
Tried out a fnapi and non-fnapi job to see that it works and optimizations is preserved respectively. Switched the fnapi wordcount to use the combiner api for testing purposes.

R: @robertwb

### CODE REVIEW

**Comment by lukecwik** [comment]
CC: @youngoli

File: sdks/python/apache_beam/runners/dataflow/dataflow_runner.py [hunk 1]
```diff
@@ -614,6 +615,15 @@ def _pardo_fn_data(transform_node, get_label):
             transform_node.inputs[0].windowing)
 
   def apply_CombineValues(self, transform, pcoll):
+    # TODO(BEAM-2937): Disable combiner lifting for fnapi. Remove this
+    # restrictions once this feature is supported in the portability framework.
```
Comment (robertwb) [inline sdks/python/apache_beam/runners/dataflow/dataflow_runner.py]: s/portability framework/dataflow runner harness/

File: sdks/python/apache_beam/runners/dataflow/dataflow_runner.py [hunk 2]
```diff
@@ -614,6 +615,15 @@ def _pardo_fn_data(transform_node, get_label):
...
+    debug_options = pcoll.pipeline._options.view_as(DebugOptions)
+    use_fn_api = (
+        debug_options.experiments and 'beam_fn_api' in debug_options.experiments
```
Comment (robertwb) [inline sdks/python/apache_beam/runners/dataflow/dataflow_runner.py]: I assume this is inferred when streaming is set?

File: sdks/python/apache_beam/runners/dataflow/dataflow_runner.py (hunk 1 above)
Comment (aaltay) [reply sdks/python/apache_beam/runners/dataflow/dataflow_runner.py]: Done.

File: sdks/python/apache_beam/runners/dataflow/dataflow_runner.py (hunk 2 above)
Comment (aaltay) [reply sdks/python/apache_beam/runners/dataflow/dataflow_runner.py]: Thank you for catching this. It is inferred in the apiclient, but it is inferred after the graph was built.

**Pull request merged**